migration/runner.go

```diff
@@ -112,7 +110,7 @@ func RunMigration(ctx context.Context, cfg Config, cache MigrationCache, store c
 					rate := float64(doneNow) / elapsed.Seconds()
 
 					log.Log(rt.INFO, "Performing migration: %d of %d jobs processed (%.0f/s) [%v elapsed]",
-						doneNow, jobsNow, rate, elapsed.Round(time.Second))
+						doneNow, jobsNow, rate, elapsed.Round(100*time.Millisecond))
```

Comment:
let's leave this one as seconds, it's likely coming out on a seconds cadence anyway and nobody's using this for benchmarking

```suggestion
						doneNow, jobsNow, rate, elapsed.Round(time.Second))
```